packages/espressocash_api/lib/src/dto/gas.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'gas.freezed.dart';
+part 'gas.g.dart';
+
+@freezed
+class GasFeeRequestDto with _$GasFeeRequestDto {
+  const factory GasFeeRequestDto({
+    required String network,
+  }) = _GasFeeRequestDto;
+
+  factory GasFeeRequestDto.fromJson(Map<String, dynamic> json) =>
+      _$GasFeeRequestDtoFromJson(json);
+}
+
+@freezed
+class GasFeeResponseDto with _$GasFeeResponseDto {
+  const factory GasFeeResponseDto({
+    required GasFeeEstimate low,
+    required GasFeeEstimate medium,
+    required GasFeeEstimate high,
+    required String estimatedBaseFee,
+    required double networkCongestion,
+    required List<String> latestPriorityFeeRange,
+    required List<String> historicalPriorityFeeRange,
+    required List<String> historicalBaseFeeRange,
+    required String priorityFeeTrend,
+    required String baseFeeTrend,
```

Comment:
Will all these fields be used in the client?

Reply:
we only need this 

`Suggested gas fees: {
  low: {
    suggestedMaxPriorityFeePerGas: '0.05',
    suggestedMaxFeePerGas: '28.501086221',
    minWaitTimeEstimate: 15000,
    maxWaitTimeEstimate: 30000
  },
  medium: {
    suggestedMaxPriorityFeePerGas: '0.1',
    suggestedMaxFeePerGas: '38.508966399',
    minWaitTimeEstimate: 15000,
    maxWaitTimeEstimate: 45000
  },
  high: {
    suggestedMaxPriorityFeePerGas: '0.3',
    suggestedMaxFeePerGas: '48.666846576',
    minWaitTimeEstimate: 15000,
    maxWaitTimeEstimate: 60000
  },
  estimatedBaseFee: '28.451086221',
  networkCongestion: 0.7718,`

Reply:
updated endpoints